src/hltas.cpp

```diff
@@ -614,34 +618,47 @@ namespace HLTAS
 						char *s2;
 						f.X = std::strtod(s, &s2); // TODO: replace this with lexical_cast probably.
 						f.Y = std::strtod(s2 + 1, nullptr);
+					} else if(str == "*") {
+						f.Yaw = lastYaw;
```

Comment:
Well, what happens if there's no last yaw? That's right, undefined behavior, which is bad. Make `lastYaw` a `std::optional` and make it an error if it's not present.